app/views/audits/audits/no_content_to_audit.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :sidebar do %>
+  <%= render 'sidebar' %>
+<% end %>
+
+<% content_for :header do %>
+  <%= render 'audits/common/navigation' %>
+<% end %>
+
+<div class="alert alert-info" role="alert">
+  <p>
+    <%= message -%>
+  </p>
+  <% unless params[:allocated_to].blank? || params[:allocated_to] == Audits::Policies::Allocated::NO_ONE.to_s %>
```

Comment:
I would extract this into a business rule. Some that can be reuse. It seems related to the 'Filter'